.github/workflows/full-test-suite.yaml

```diff
@@ -51,6 +59,12 @@ on:
       AZURE_APP_SECRET:
         description: ''
         required: true
+      GH_FRONTEGG_BOT_APP_ID:
```

Comment:
So here we might have an issue,
this is from oauth-service
```
run-full-test:
    name: Run E2E + API Tests (venv)
    if: github.event.pull_request.draft == false
    needs: [ main-pull-request, prettier-check ]
    uses: frontegg/workflows/.github/workflows/full-test-suite.yaml@master
    with:
      build_image: true
    secrets: inherit
    ```
    
As you can see the trigger is with `secrets: inherit` which solve the issue but if a workflow does not use inherit we will have an issue. You need to tell everyone that you are making this change and they need to pay attention

Reply:
if the AZURE_APP_SECRET is organization secret then every one use it with secrets inheret.
I know that GH_FRONTEGG_BOT_APP_ID is always there because it's organization secret.

correct me if I'm wrong